ocs_ci/ocs/scale_lib.py

```diff
@@ -649,3 +651,49 @@ def check_and_add_enough_worker(worker_count):
             raise UnavailableResourceException(
                 "There is no enough worker nodes to continue app pod scaling"
             )
+
+
+def increase_pods_per_worker_node_count(count=500):
+    """
+    Function to increase pods per node count, default OCP supports 250 pods per node,
+    from OCP 4.6 limit is going to be 500, but using this function can override this param
+    to create more pods per worker nodes.
+    more detail: https://docs.openshift.com/container-platform/4.5/nodes/nodes/nodes-nodes-managing-max-pods.html
+    WARN: This function will perform Unscheduling of workers and back to Scheduled state
+    Please aware if there is any non-dc pods then expected to be terminated.
+
+    Args:
+        count (int): Required pods per node count
+
+    """
+    max_pods_template = templating.load_yaml(constants.PODS_PER_NODE_COUNT_YAML)
+    max_pods_template['spec']['kubeletConfig']['podsPerCore'] = 65
+    max_pods_template['spec']['kubeletConfig']['maxPods'] = count
+
+    # Create new max-pods label
+    max_pods_obj = OCS(**max_pods_template)
+    assert max_pods_obj.create()
+
+    # Apply the changes in the workers
+    label_cmd = "label machineconfigpool worker custom-kubelet=small-pods"
+    ocp = OCP()
+    assert ocp.exec_oc_cmd(command=label_cmd)
+
+    # First wait for Updating status to become True, default it will be False &
+    # machine_count and ready_machine_count will be equal
+    get_cmd = "get machineconfigpools -o yaml"
+    while True:
+        output = ocp.exec_oc_cmd(command=get_cmd)
+        update_status = output.get('items')[1].get('status').get('conditions')[4].get('status')
+        if update_status == 'True':
+            break
```

Comment:
Can you add another count variable that can break from loop after some count ?

Reply:
ack